src/main/java/com/rick/racing/model/CarRecordHistory.java

```diff
@@ -0,0 +1,43 @@
+package com.rick.racing.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarRecordHistory {
+
+    public static final int GO_VALUE = 1;
+    public static final int STOP_VALUE = 0;
+
+    final private List<Integer> positions = new ArrayList<>();
```

Comment:
```suggestion
    private final List<Integer> positions = new ArrayList<>();
```
프로그래밍 요구사항의 `자바 코드 컨벤션`을 한 번 더 확인해주세요.
https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers